### PR TITLE
port changed

### PR DESCRIPTION
**Pull request by noorrozgar**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,6 @@ app.get('/', (req, res) => {
 res.send('<h1><center>Nodejs app workig<center></h1>');
 });
 
-app.listen(3000, () => {
```
Comment (noor1992) [inline index.js:8]: can u change port to 3001

Comment (noorrozgar) [reply index.js:8]: Yes I can

**Review by noorrozgar — COMMENTED** [review]
port changed to 3001

**Review by noor1992 — APPROVED** [review]
Looks Good

**Pull request merged**